### DISCUSSION TITLE
ionic build issue after change app bundle ID [10:02:38]: fastlane finished with errors /usr/local/bundle/bin/fastlane: \e[31m[!] undefined method `content=' for nil:NilClass\e[0m (NoMethodError)

### DISCUSSION THREAD

**Discussion by pollytoy** [fastlane/fastlane#20248] [Q&A]
+----------------------------+----------------------------------------------------------------+
|                                        Lane Context                                         |
+----------------------------+----------------------------------------------------------------+
| DEFAULT_PLATFORM           | ios                                                            |
| PLATFORM_NAME              | android                                                        |
| LANE_NAME                  | android package_build                                          |
| PROJECT_WEB_DIR            | www                                                            |
| DOWNLOAD_CREDENTIAL_EXISTS | true                                                           |
| DOWNLOAD_CREDENTIAL_PATH   | /builds/dexycon/beer.kolobars.bar.dexycon.com/android.keystore |
| DOWNLOAD_CREDENTIAL_ALIAS  | EndeavorApp                                                    |
+----------------------------+----------------------------------------------------------------+

+------+-------------------------+-------------+
|               fastlane summary               |
+------+-------------------------+-------------+
| Step | Action                  | Time (in s) |
+------+-------------------------+-------------+
| 1    | default_platform        | 0           |
| 2    | build_summary           | 0           |
| 3    | add_git_credentials     | 0           |
| 4    | get_appflow_config      | 0           |
| 5    | detect_native_type      | 0           |
| 6    | dependency_install      | 189         |
| 7    | create_capacitor_config | 0           |
| 8    | get_web_dir             | 0           |
| 9    | set_java_home           | 0           |
| 10   | modify_cap_web_config   | 0           |
| 11   | download_credential     | 1           |
| 12   | build_pro_app           | 134         |
| 13   | cap_sync                | 2           |
| 14   | cap_custom_deploy       | 0           |
| 💥   | build_android_capacitor | 0           |
| 16   | upload_logs             | 5           |
+------+-------------------------+-------------+

[10:02:38]: fastlane finished with errors
/usr/local/bundle/bin/fastlane: \e[31m[!] undefined method `content=' for nil:NilClass\e[0m (NoMethodError)
Running after script...
$ run "clean-up"
ERROR: Job failed: command terminated with exit code 1

if anyone have idea how to fix this please to help me.

**Comment by PacoRivera13** [comment]
> +----------------------------+----------------------------------------------------------------+ | Lane Context | +----------------------------+----------------------------------------------------------------+ | DEFAULT_PLATFORM | ios | | PLATFORM_NAME | android | | LANE_NAME | android package_build | | PROJECT_WEB_DIR | www | | DOWNLOAD_CREDENTIAL_EXISTS | true | | DOWNLOAD_CREDENTIAL_PATH | /builds/dexycon/beer.kolobars.bar.dexycon.com/android.keystore | | DOWNLOAD_CREDENTIAL_ALIAS | EndeavorApp | +----------------------------+----------------------------------------------------------------+
> 
> +------+-------------------------+-------------+ | fastlane summary | +------+-------------------------+-------------+ | Step | Action | Time (in s) | +------+-------------------------+-------------+ | 1 | default_platform | 0 | | 2 | build_summary | 0 | | 3 | add_git_credentials | 0 | | 4 | get_appflow_config | 0 | | 5 | detect_native_type | 0 | | 6 | dependency_install | 189 | | 7 | create_capacitor_config | 0 | | 8 | get_web_dir | 0 | | 9 | set_java_home | 0 | | 10 | modify_cap_web_config | 0 | | 11 | download_credential | 1 | | 12 | build_pro_app | 134 | | 13 | cap_sync | 2 | | 14 | cap_custom_deploy | 0 | | 💥 | build_android_capacitor | 0 | | 16 | upload_logs | 5 | +------+-------------------------+-------------+
> 
> [10:02:38]: fastlane finished with errors /usr/local/bundle/bin/fastlane: \e[31m[!] undefined method `content=' for nil:NilClass\e[0m (NoMethodError) Running after script... $ run "clean-up" ERROR: Job failed: command terminated with exit code 1
> 
> if anyone have idea how to fix this please to help me.

Were you able to solve your issue?
